com.jetbrains.PyCharm

```diff
@@ -0,0 +1,94 @@
+{
+    "app-id": "com.jetbrains.PyCharm",
```

Comment:
Since there is a difference between PyCharm and the community edition maybe this should have a suffix: `com.jetbrains.PyCharm-ce` or such?

Reply:
Yeah, we did this with Visual Studio Code as well (since the OSS version declares a different app id). I think that's worthwhile.

Reply:
Yeah, this is really important note. I will rename it and may be also the JSON itself